aten/src/ATen/native/transformers/attention.cpp

```diff
@@ -683,10 +687,15 @@ Tensor scaled_dot_product_attention(
         auto out_lse_softmax = at::_scaled_dot_product_flash_attention(
             query_padded, key_padded, value_padded, dropout_p, is_causal, false /*return_debug_mask*/, og_scale.as_float_unchecked());
         return post_process_flash_output(std::get<0>(out_lse_softmax), og_size);
-      }
+      } else if (query_.device().type() == DeviceType::PrivateUse1) {
```

Comment:
Can it be enabled for other backend devices too, so that out of tree backends can register the _scaled_dot_product_fused_attention_overrideable and the corresponding bwd op kernels?